docs/development/update-helm-charts.md

```diff
@@ -0,0 +1,52 @@
+# Update helm charts for EKS-A Packages
+
+Helm charts used in EKS-A Packages are required to go through a series of modification to meet release standards. This file documents some standard modification procedures that apply to (almost) all packages. Note helm chart structure varies by repo, so use judgment while applying following changes.
+
+## Generate patch files
+
+Helm charts modifications are done through patches. To do so, we perform the following procedures:
+- clone the target helm chart repo locally;
+- update helm charts locally (see details at next step); and 
+- generate patch files using [`git format-patch`](https://git-scm.com/docs/git-format-patch).
+
+## Update helm charts locally
+### Update `values.yaml` file
+
+Following changes need to be made to the `values.yaml` file:
+
+- `sourceRegistry`
+    - Add this field with value `public.ecr.aws/eks-anywhere`. 
+    - To test the helm chart locally, you can call `helm install` with flag `--set sourceRegistry=${YourECRRegistry}` to override its value.
+- `image:repository` 
```

Comment:
`image.repository` won't always be at the root. In the example you wrote, `metallb/controller` and `metallb/speaker` are in a single helm chart, not multiple. Each have their own sections where you can set the image repository. I'd instead write that the image registry and repository should be configured via `values.yaml` to allow overriding and that `digest` is set in `values.yaml` and replaced automatically by the build process.

Reply:
Thanks for pointing this out! I updated the example description of metallb to be more accurate. I also added (1) a note on the top of this doc that helm chart repo structure varies, and (2) modified the field descriptions to be more open-ended.